src/SignalR/common/Http.Connections/src/Internal/Transports/LongPollingServerTransport.cs

```diff
@@ -11,17 +11,17 @@
 
 namespace Microsoft.AspNetCore.Http.Connections.Internal.Transports
 {
-    internal class LongPollingTransport : IHttpTransport
+    internal class LongPollingServerTransport : IHttpTransport
     {
         private readonly PipeReader _application;
         private readonly ILogger _logger;
         private readonly CancellationToken _timeoutToken;
 
-        public LongPollingTransport(CancellationToken timeoutToken, PipeReader application, ILoggerFactory loggerFactory)
+        public LongPollingServerTransport(CancellationToken timeoutToken, PipeReader application, ILoggerFactory loggerFactory)
         {
             _timeoutToken = timeoutToken;
             _application = application;
-            _logger = loggerFactory.CreateLogger<LongPollingTransport>();
+            _logger = loggerFactory.CreateLogger("LongPollingTransport");
```

Comment:
I believe it's supposed to be "Microsoft.AspNetCore.Http.Connections.Internal.Transports.LongPollingTransport" ditto for the others

Reply:
Yeah, should be the full name